0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
I mean, they probably copied it from `rust-lang/rust`, e.g. https://github.com/rust-lang/rust/blob/master/LICENSE-MIT#L1. Is that wrong too?